cli/pcluster/cli_commands/start.py

```diff
@@ -121,9 +122,10 @@ def start(self, args, pcluster_config):
             error("Failed when starting compute fleet with error: {}".format(e))
 
 
-SCHEDULER_TO_START_COMMAND_MAP = {
-    "awsbatch": AWSBatchStartCommand,
-    "sge": SITStartCommand,
-    "torque": SITStartCommand,
-    "slurm": HITStartCommand,
-}
+def get_start_command_for_stack(stack, scheduler):
+    if is_hit_enabled_cluster(stack):
```

Comment:
Why not moving this code to cluster_model? Then we could do `pcluster_config.cluster_model.get_start_command()`

Reply:
done